packages/block-library/src/common.scss

```diff
@@ -32,16 +32,17 @@
 }
 
 // Text alignments.
-.has-text-align-center {
+// Increase specificity so that styles applied via the global styles can be overridden in block instances.
+:root .has-text-align-center {
```

Comment:
Do we not need the same for `.has-larger-font-size` etc. above?

Reply:
Honestly, it sounds to me like these types of selectors should move _after_ theme.json styles. These are not default styles to me.